libcontainer/cgroups/fs/memory.go

```diff
@@ -209,6 +217,13 @@ func (s *MemoryGroup) GetStats(path string, stats *cgroups.Stats) error {
 	if value == 1 {
 		stats.MemoryStats.UseHierarchy = true
 	}
+
+	pagesByNUMA, err := getPageUsageByNUMA(path)
+	if err != nil {
+		return err
```

Comment:
You could add some info about from which part this error is going, as in lower level you have e.g. i/o errors mixed with custom string errors.